poetry/patterns/header.php

```diff
@@ -12,8 +12,8 @@
 <div class="wp-block-group"><!-- wp:site-title {"level":0,"isLink":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"600"}},"fontSize":"medium"} /-->
 
 <!-- wp:group {"style":{"spacing":{"blockGap":"48px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
-<div class="wp-block-group"><!-- wp:paragraph {"style":{"typography":{"lineHeight":"0.8"}},"fontSize":"xx-large","fontFamily":"rozha-one"} -->
-<p class="has-rozha-one-font-family has-xx-large-font-size" style="line-height:0.8">Food <br>for thoughts</p>
+<div class="wp-block-group"><!-- wp:paragraph {"style":{"typography":{"lineHeight":"0.8"}},"fontSize":"xx-large","fontFamily":"heading"} -->
+<h2 class="has-heading-font-family has-xx-large-font-size" style="line-height:0.8">Food <br>for thoughts</h2>
```

Comment:
we made this change but then we don't assign a "heading" font family in the .json files. The class `has-heading-font-family` is not resolving to anything so this change is not really doing anything. We could remove the class altogether and just let the theme default to whatever the theme.json sets for all headings. That is probably the best case for this particular theme, but that means that no templates/patterns can select specific font families and expect the theme to be able to adapt when we change to a different variation that doesn't have that specific font family.